app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -96,11 +96,13 @@ def successful_result?
         end
 
         def error_messages
-          return {} if successful_result?
-
-          {
-            socure: { reason_codes: get_data(DATA_PATHS[:reason_codes]) },
-          }
+          if !successful_result?
+            { socure: { reason_codes: get_data(DATA_PATHS[:reason_codes]) } }
+          elsif !pii_valid?
+            { pii_validation: 'failed' }
```

Comment:
we could call this like, `local_attribute_validation` or something and not have to worry about having `pii` in the key name?

Reply:
I kinda prefer the clarity, since we have a ticket coming up shortly to actually display more specific errors about pii, and would likely want to use `pii*` keys anyway; might as well cope with it now.